### DISCUSSION TITLE
Error when auth between github and heroku app

### DISCUSSION THREAD

**Discussion by FatmaMahmoud698** [community/community#21989] [New to GitHub]
<p>I tried to connect between the github repository and heroku app and I get this error<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/2/e21dcb4a63e7b51bbd2bbecbd8a5020a84f4e47e.png" data-download-href="/uploads/short-url/wgjJZQfRvlSea5uZhB6AtDb2zgi.png?dl=1" title="1" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75572871/181068059-d9494af6-2878-4c92-aaa6-0a8d55090ff3.png" alt="1" data-base62-sha1="wgjJZQfRvlSea5uZhB6AtDb2zgi" width="690" height="319" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e21dcb4a63e7b51bbd2bbecbd8a5020a84f4e47e_2_690x319.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e21dcb4a63e7b51bbd2bbecbd8a5020a84f4e47e_2_1035x478.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e21dcb4a63e7b51bbd2bbecbd8a5020a84f4e47e_2_1380x638.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e21dcb4a63e7b51bbd2bbecbd8a5020a84f4e47e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">1</span><span class="informations">1910×884 79.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/fatmamahmoud698">@FatmaMahmoud698</a>,</p>
<p>Thanks for being a part of the GitHub community! We’re glad you’re here.</p>
<p>I wanted to jump in and mention that if you’re looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Heroku, like their <a href="https://stackoverflow.com/questions/tagged/heroku">Stack Overflow Community</a>. It’s definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Forum focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this forum may not be the right place for this particular topic.</p>
<p>Best of luck!</p>

**Comment by airtower-luna** [comment]
<p>Note that there has been a recent security incident involving leaked Heroku tokens. That could cause issues here, too, depending on details.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.blog/2022-04-15-security-alert-stolen-oauth-user-tokens/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/5814919/181068061-ac507dd2-cdd4-4813-b2e9-e041e3078ee1.png" class="site-icon" width="32" height="32">

      <a href="https://github.blog/2022-04-15-security-alert-stolen-oauth-user-tokens/" target="_blank" rel="noopener" title="10:35PM - 15 April 2022">The GitHub Blog – 15 Apr 22</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/5814919/181068062-c6e02713-9318-4e4a-987f-40e892a29212.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2cee7d85cf2fc8378c296e0b5c7ff1874954bc8_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2cee7d85cf2fc8378c296e0b5c7ff1874954bc8_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2cee7d85cf2fc8378c296e0b5c7ff1874954bc8_2_1380x724.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2cee7d85cf2fc8378c296e0b5c7ff1874954bc8_2_10x10.png"></div>

<h3><a href="https://github.blog/2022-04-15-security-alert-stolen-oauth-user-tokens/" target="_blank" rel="noopener">Security alert: Attack campaign involving stolen OAuth user tokens issued to...</a></h3>

  <p>On April 12, GitHub Security began an investigation that uncovered evidence that an attacker abused stolen OAuth user tokens issued to two third-party OAuth integrators, Heroku and Travis-CI, to download data from dozens of organizations, including...</p>

  <p>
    <span class="label1">Est. reading time: 5 minutes</span>
  </p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
